api/serializers.py

```diff
@@ -5,10 +5,14 @@
 from rest_framework_json_api import serializers
 
 from share import models
-from share.models import ProviderRegistration, SiteBanner, CeleryTaskResult
 
-from api import fields
 
+from share.models import ProviderRegistration, SiteBanner, CeleryTaskResult, HarvestLog, SourceConfig
+
+
+from share.models import ProviderRegistration, SiteBanner, CeleryTaskResult, logs
```

Comment:
Remove all the `from share.models` imports, just use `from share import models` (and update existing things to match)